README.md

```diff
@@ -10,3 +10,13 @@
 # Work in progress
 
 Sketching out a reliable binary provider!
+
+# `@staticfloat`'s initial draft
+
+Borrowing inspiration from [Homebrew](https://github.com/Homebrew/brew/), this
+initial draft unpacks archives into separate folders within a `staging` area,
+then symlinks the resulting files into a `prefix`.
```

Comment:
To try and solve the uninstallation problem; I've switched over to trying to determine installation receipts by asking our unpacking tools to give me a file manifest instead.  I didn't realize that Windows doesn't let you make symlinks unless you're an Administrator.  🦆